.buildkite/packaging.pipeline.yml

```diff
@@ -265,6 +334,76 @@ steps:
         artifact_paths:
           - build/distributions/**/*
 
+      - label: "STAGING: {{matrix}} FIPS"
```

Comment:
```suggestion
      - label: "STAGING: {{matrix}} Linux/amd64 FIPS"
```

Reply:
Fixed in e1cf0bd091e854c0a8a936fe6dfb234c0cea98ad